examples/proxy/record-encryption/README.md

```diff
@@ -0,0 +1,22 @@
+# AMQ Streams Proxy Record Encryption Filter Examples
+
+The Encryption filter which provides an Encryption-at-Rest solution for Apache Kafka(tm) which is transparent to both clients and brokers. The filter takes
```

Comment:
nit? the filter is the `RecordEncryption` so `The Encryption filter` feels a bit of an ambiguous reference.